packages/x-data-grid-premium/src/models/dataGridPremiumProps.ts

```diff
@@ -124,6 +125,16 @@ export interface DataGridPremiumPropsWithDefaultValue<R extends GridValidRowMode
    * @default (pastedText) => { const text = pastedText.replace(/\r?\n$/, ''); return text.split(/\r\n|\n|\r/).map((row) => row.split('\t')); }
    */
   splitClipboardPastedText: (text: string) => string[][] | null;
+  /**
+   * If `true`, the AI Assistant panel is open.
+   * @default false
+   */
+  aiAssistantPanelOpen: boolean;
+  /**
+   * If `true`, the AI Assistant is disabled.
+   * @default false
+   */
+  disableAiAssistant: boolean;
```

Comment:
```suggestion
  /**
   * If `true`, the AI Assistant is enabled.
   * @default false
   */
  enableAiAssistant: boolean;
```

we need the opposite here, since you can't just have this feature.
additional steps are needed (connection to the backend). once you have them, you can enable the feature for your grid